### DISCUSSION TITLE
Dependencies

### DISCUSSION THREAD

**Discussion by dalekvim** [PrajaktaSathe/Python#15] [Ideas]
I want to contribute to your project:

### What I want to add:

- An easy way to install **dependencies**.
- **Online documentation** using sphinx and GitHub pages.
- Instructions on how to **set up** the repository for newbies.

Do you think I should place all the dependencies in the root of the folder, or separately for each of the projects?
I would recommend the second option.

Thank you for hearing me out, hope we can start working together soon!

**Comment by PrajaktaSathe** [comment]
Hello @Dalekvim !!
I read your ideas, and I think you can go ahead with all of them! They will definitely be a nice addition to the repository!
